saleor/csv/utils/export.py

```diff
@@ -60,6 +61,32 @@ def get_filename(model_name: str, file_type: str) -> str:
     )
 
 
+def parse_input(data: Any) -> Dict[str, Union[str, dict]]:
+    """Parse input to correct data types, since scope coming from celery will be parsed to strings."""
+    if "attributes" in data:
+        serialized_attributes = []
+
+        for attr in data.get("attributes") or []:
```

Comment:
Can't it be `data.get("attributes", []):` ?

Reply:
I would leave coz I'm not sure if there is a case whrere attributes might be none.